packages/web/pages/player/[username].tsx

```diff
@@ -87,58 +90,81 @@ export const PlayerPage: React.FC<Props> = ({
   if (!player) return <Page404 />;
 
   return (
-    <PageContainer p={0} px={[0, 4, 8]}>
+    <PageContainer pt={0} px={[0, 4, 8]}>
       <HeadComponent
-        title={`MetaGame Player Profile: ${getPlayerName(player)}`}
+        title={`MetaGame Profile: ${getPlayerName(player)}`}
         description={(getPlayerDescription(player) ?? '').replace('\n', ' ')}
         url={getPlayerURL(player, { rel: false })}
         img={getPlayerImage(player)}
       />
       <Box
-        bg={`url(${getPlayerBannerFull(player)}) no-repeat`}
+        bg={`url(${banner}) no-repeat`}
         bgSize="cover"
         bgPos="center"
         h={72}
         pos="absolute"
         w="full"
+        top={0}
       />
       <Flex w="full" h="full" pt="3rem" direction="column" align="center">
-        <Grid {...{ player, personalityInfo }} />
+        <Grid {...{ player }} />
       </Flex>
     </PageContainer>
   );
 };
 
 export default PlayerPage;
 
-const getBoxKeyFromTarget = (target: HTMLElement | null): string =>
-  (target?.offsetParent as HTMLElement)?.offsetParent?.id ?? '';
+const makeLayouts = (editing: boolean, layouts: Layouts): Layouts =>
+  Object.fromEntries(
+    Object.entries(layouts).map(([key, items]) => [
+      key,
+      items.map((item) =>
+        item.i === 'hero' ? { ...item, isResizable: editing } : item,
```

Comment:
This seems like a rebase error to me. This method is not necessary and not used in `develop` branch

Reply:
All these helpers were moved to `utils/layoutHelpers.ts`

Reply:
Removed from `[tsername].tsx`.